examples/platform_channel_swift/README.md

```diff
@@ -0,0 +1,16 @@
+# Example of calling platform services from Flutter
+
+This project demonstrates how to connect a Flutter app to platform
+specific services on iOS using Swift. The equivalent version of this
+project in Objective C is found in flutter/examples/platform_channel.
```

Comment:
I would change the path to `/examples/platform_channel/` to avoid confusion over what the install folder might be called